test-files/cases/attribute/unifying/highway-2927-bridge-1/Expected.osm

```diff
@@ -3818,25 +3818,27 @@
         <nd ref="-595"/>
         <nd ref="-563"/>
         <nd ref="-564"/>
+        <nd ref="-611"/>
```

Comment:
Unfortunately, this one of two test outputs that got slightly worse. The difference is fairly minor (comments in the README), so decided to go with it after not being able to prevent it after the fact.